.github/workflows/ci-tests.yml

```diff
@@ -105,6 +112,12 @@ jobs:
 
   LightHouse:
     name: Test site performance using Lighthouse
+
+    # Don't run action if the pull request is a draft.
+    # This is an expensive job to run, so we only want to run it when the
+    # PR is actually ready to be reviewed.
+    if: ${{ github.event.pull_request.draft == false }}
+
```

Comment:
Personally I am not fan of this change.

My workflow is usually that I initially create pull requests on GitHub in the draft state to indicate that it is not ready for review (or merge-able), see that al checks are green and then assign it. Now I will have to create the PRs that are ready for review before I can see if all checks are green. 

Alternately I can create them as draft, see that they are green, then mark them as ready for review, then I have to see that they are still green. *Then* I can assign them.

Perhaps not a huge problem - but different and potentially more time consuming.

I